base.docker

```diff
@@ -4,6 +4,10 @@ ENV APP_DIR /app
 
 ENV NODE_VERSION 6.12.2
 ENV YARN_VERSION 1.3.2-1
+ENV SUPERVISOR_VERSION 3.3.3
+ENV UWSGI_VERSION 2.0.15
+ENV AWSCLI_VERSION 1.14.31
+ENV AWSCLI_CWLOGS_VERSION 1.4.4
```

Comment:
Something in _this_ section, *unbelieveable* (to me at least), has broken this... 😕  Investingating which one in particular...